pegasus/sites.v3/hourofcode.com/styles/050-theme.css

```diff
@@ -18,7 +18,7 @@ body {
 
 #pageheader {
   width: 100%; 
-  background-color: #000;
+  background-color: rgba(0, 178, 192, 0.5);
```

Comment:
Okay, just tested this. It indeed does not work in IE8. Looks like the stylesheet sets the "fallback" color (#000) and then the transparent rgba color is set inline in the html. Fixing this with my next check-in.